config/src/main/java/tech/pegasys/pantheon/config/CliqueConfigOptions.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2018 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.pantheon.config;
+
+import io.vertx.core.json.JsonObject;
+
+public class CliqueConfigOptions {
+
+  public static final CliqueConfigOptions DEFAULT = new CliqueConfigOptions(new JsonObject());
+
+  private static final long DEFAULT_EPOCH_LENGTH = 30_000;
```

Comment:
tbh - I really hate default values for this - people should configure their network to actually work - not just get lucky that things came out well.
These defaults would need to become part of the clique spec to ensure unconfigured clients would interact correctly ...

Reply:
I don't mean this to sound defensive, but I didn't introduce those defaults, you did. I just moved them. :)

In general though I think it makes a lot of sense to have defaults for these config values and it seems like pretty common behaviour in other clients.  The spec does explicitly recommend 30000 block epoch length.

Reply:
Yeah, I put them in because other people did ... the defaults do now form part of our spec, maybe not a bad thing.